recipes/napari-annotate/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "napari-annotate" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-annotate-{{ version }}.tar.gz
+  sha256: 37c89c9a8b7c877ae254e7a45af581cf3787e47549b292a10fd40a9da0be1406
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - magic-class
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - napari-tools-menu
+    - numpy
+    - python >=3.8
+
+test:
+  imports:
+    - napari_annotate
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/WyssCenter
```

Comment:
Not sure why this is the site listed since that org doesn't show a repo for the plugin anywhere, but it matches what's on PyPI.